diff --git a/pipeline_test.py b/pipeline_test.py
index c4fccc16f30dc6af17ccbdb3f70a9b653cab3264..df78c4015d1065b5c84c90103508c159aa2a42a0 100644
--- a/pipeline_test.py
+++ b/pipeline_test.py
@@ -36,38 +36,38 @@ def load_json():
 
 def main():
     centre = load_json()
 
     video = load()
     frames, height, width = video.shape
     # play_video_cv2(video)
 
     bkg = np.median(video[:20, :, :], axis = 0)
 
     sub_bkg = video - bkg[None, :, :]
 
 
     lap5x5 = np.array([
         [-1, -3, -4, -3, -1],
         [-3,  0,  6,  0, -3],
         [-4,  6, 20,  6, -4],
         [-3,  0,  6,  0, -3],
         [-1, -3, -4, -3, -1],
     ], dtype=np.float32)
 
     # video_lap = filter_video_fft(video, lap5x5)
     # play_videos_side_by_side((video_lap, video))
 
     video_lap = filter_video_fft(sub_bkg, lap5x5)
-    video_lap= np.abs(video_lap)
-    video_lap_med = median_filter_video_cuda(video_lap, 3, 3, 3)
+    video_lap = np.abs(video_lap)
+    video_lap_med = median_filter_video_auto(video_lap, 3, 3, 3)
     play_videos_side_by_side((video_lap_med, video_lap, sub_bkg, video))
 
     bins, signal, density = angle_signal_density(video_lap_med, centre[0], centre[1], 3600)
     plot_angle_signal_density(bins, signal, log=True, plume_angles=None)
     
     bw, thres = triangle_binarize(signal)
     plt.imshow(bw, origin="lower")
 
 
 if __name__ == '__main__':
     main()
